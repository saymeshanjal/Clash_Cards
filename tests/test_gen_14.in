51
10 93 2
4 31 1
14 6 4
8 90 1
10 9 4
4 18 2
2 46 4
10 53 4
2 72 2
8 97 5
12 69 3
14 39 2
12 29 3
2 28 5
4 65 4
6 10 4
10 82 2
2 66 1
2 39 5
4 82 2
8 87 4
16 66 3
6 75 4
16 67 3
6 84 5
6 47 5
2 35 4
2 24 5
14 20 4
2 5 1
10 65 4
12 92 5
12 22 2
6 9 4
2 95 5
8 41 2
10 17 5
8 69 4
12 4 1
4 92 2
10 32 4
6 97 3
6 74 5
4 45 1
16 89 1
12 73 4
16 9 3
2 77 2
4 98 1
6 7 4
12 87 3
