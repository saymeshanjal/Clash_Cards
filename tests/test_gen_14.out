76 77 99
3 3 0
8 3158 266
35 35 0
0 235 17
1 2 3
0 2 0
44 133 75
0 1 1
8 62 0
430 494 0
229 408 2795
167 757 0
0 2 0
0 5 1
0 18 2
76 77 99
1 1 0
0 2 0
1 2 3
8 42 20
5368 7502 0
1 14 5
5463 7407 0
1 19 0
1 19 0
0 2 0
0 2 0
54 2989 389
1 1 0
44 133 75
219 705 0
27 106 791
0 18 2
0 2 0
20 21 29
4 248 0
8 42 20
336 588 0
1 2 3
20 137 95
6 14 0
1 19 0
3 3 0
6435 6435 0
208 441 275
1169 11701 0
0 1 1
3 3 0
0 18 2
430 494 0
